### ISSUE TITLE
Question about msgs.. how to?

### ISSUE THREAD

**Issue by abixadamj** [blockpy-edu/blockpy#34]
I need some help.
I have install blockpy and change en.js to pl.js (for Polish language)
Now I have blocks in Polish language, but I still have sections (like Variables) in English
![image](https://user-images.githubusercontent.com/5088643/37865023-f7d4b57a-2f76-11e8-9c3d-5aeedd4932d7.png)

Anyone can help me with that issue? I have no idea, where can I change it. In html I changed <script> ... /en.js to pl.js
I have added lang=pl to html header. Still does not work.

lease, help me.
Adam

**Comment by avvalcode** [comment]
same here
I want to translate or change the language. can someone help?

**Comment by abixadamj** [comment]
No one could help?

**Comment by acbart** [comment]
Hi, here's a message I wrote on the forums a while back:

Interesting question! The blockly interface library support multiple languages. However, we have not been using that library in such a way that it would currently be convenient to create additional language versions. Presently, there are no development resources available among the team to make the fixes necessary. Perhaps if there is more interest from the community, such features can become higher on our priority list. I believe there were talks about a korean version at one point, but that was a while ago

Basically, we don't have the time and resources to support this feature, as cool as it would be. We'd be happy to review Pull Requests, but can't commit to anything more substantial than that. Very sorry!

**Comment by acbart** [comment]
If this gets attention from anyone, I'd be interested in hearing whether we need to translate:

1) The whole blockpy interface
2) Just the blocks
3) Just student-facing parts of the interface

Essentially, I'd like some ideas on how narrow the scope should be to be useful. I'm not opposed to this, but it'll be some major rearchitecturing since currently strings are littered in many areas. A first step might be to make BlockMirror support multiple languages, before tackling the student facing side of BlockPy.
